db/1.13/modules/ROOT/pages/database-connector-examples.adoc

```diff
@@ -31,6 +31,9 @@ Configure custom data types to use when connected to a particular connection pro
 Execute database operations in the context of a transaction.
 * xref:database-insert-large-clob.adoc[Insert large file content into a table] +
 Configure a Mule app flow to read and insert a large file into a table.
+* xref:database-connector-xa-transactions.adoc[Using XA Transactions in Database Connector] +
+Configure the *Use XA Transactions* field in the Database Connector global configuration so the datasource supports Extended Architecture transactions.
```

Comment:
Later on, you have "Extended Architecture Transactions (XA Transactions)." If this is the first mention in the topic, you might want to put that here as well.